package-lock.json

```diff
@@ -33815,6 +34589,25 @@
         "@opentelemetry/api": "^1.3.0"
       }
     },
+    "packages/opentelemetry-sampler-aws-xray/node_modules/@opentelemetry/instrumentation": {
+      "version": "0.45.1",
```

Comment:
I notice that the `@opentelemetry/instrumentation` package was **not** updated for "packages/opentelemetry-sampler-aws-xray".   I think that is because it has an outdated dependency on `"@opentelemetry/contrib-test-utils": "^0.34.0"`.

https://github.com/open-telemetry/opentelemetry-js-contrib/blob/7d110d5848fe8006436ff11d269ca7e58b5e51cf/packages/opentelemetry-sampler-aws-xray/package.json#L54

That should be `"^0.35.0"`.

Why is that out of date?  It looks like that was not updated in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1798 when all other deps on `@opentelemetry/contrib-test-utils` *were* updated.

So perhaps we should update that dep first (or as part of this change) and then re-generate the update here?

Reply:
I can push a fix to this branch that does that, but we still get the `npm run compile` errors shown in the "unit-test" CI job failures.  I'm not sure what the issue is with those.